docs/google/best_practices.md

```diff
@@ -129,9 +129,18 @@ One word of caution is there is a limit to the total number of repetitions.  Tak
 that your parameter sweeps, especially products of sweeps, do not become so excessively large
 that they overcome this limit.
 
+## Use batches if sweeps are not possible
+
+The engine has a method called `run_batch()` that can be used to send multiple
+circuits in a single request.  This can be used to increase the efficiency
+and rep rate of your program.
+
+In order to efficiently batch circuits, the circuits in these batches must
+measure the same qubits and have the same number of repetitions for each
+circuit.
```

Comment:
What happens when there is a difference between measurements? How bad is it going to be? 
What amount of repetition difference are we tolerating? Can you explain a little bit more why these constraints result in the most efficient batching?

Reply:
I added a short note onto this.  However, the reasons for this are fairly deep in the hardware and I don't think it's appropriate to put too much more information on the public page.

Reply:
That sounds good - that was what I was looking for!